CHANGES.md

```diff
@@ -6,6 +6,13 @@ Version 0.6.0
 
 To be released.
 
+### Python target
+
+ -  Added `__nirum_annotations__` static field on generated union type classes.
+    [[#273] by jangjunha]
```

Comment:
Unless it's actually an all lowercase screename you intended, could you adjust the name to follow the proper orthography, i.e., Jang Junha?